_posts/2016/2016-03-24-ecommerce-barcamp-jena.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: post
+title: "Visiting the eCommerceCamp in Jena"
+date: "2016-03-24 09:03:24"
+icon: shopping-cart
+categories: events
+authors: ["Benjamin"]
+---
+
+The [eCommerceCamp](http://www.ecommerce-camp.de) is an annual [BarCamp](http://barcamp.org) that takes place in Jena, the capital of the eCommerce sphere in Germany, where a lot of the magic of online shopping started and still happens today. As always the highly affordable tickets for the unconference were sold out upfront and around 300 participants from near and far were expected. The orga team from [tritum](http://www.tritum.de) and [marmalade](http://www.marmalade.de) did a terrific job by creating an inspiring environment, which let everyone look forward to up-to-date topics about shop systems, productivity habits, email marketing, new technologies, customer satisfaction and team collaboration or even juggling with circus balls.
+
+## The participants
+
+The clientele consists mainly of UI and UX designers, frontend and backend developers, project managers, some marketing people and also a few dedicated testers from web agencies with 1 to 20 people, who typically work with self-contained content management systems like [Magento](https://magento.com), [OXID](http://www.oxid-esales.com), [Shopware](https://en.shopware.com), [TYPO3](https://typo3.org) or even [WordPress](https://wordpress.com). Attendees from the global eCommerce players are typically lesser represented, but this doesn't mean that local matadors like [Intershop](http://www.intershop.com) or ePages are not there as well.
+
+{% image blog/blog-ecommerce-barcamp-1.jpg %}{% endimage %}
+
+## The setting
+
+The event began with a dinner on Thursday evening in the famous local restaurant [Zur Noll](http://www.zur-noll.de/), which should break the ice of everyone's comfort zone and kickoff the communication flow.
+
+On Friday morning all participants met in the auditorium at the [Ernst-Abbe-Hochschule Jena](http://www.eah-jena.de/), University of Applied Sciences, which served as hosting space. After a collective breakfast the session planning for both days took place. Everybody can suggest an own Open Space talk. The crowd expresses interest in a suggestion via raising their hands. This immediate reaction allows a proper planning of the time slot and room allocation for the session.
+
+## The sessions
+
+To give you a better impression of the actual contents of the conference I want to outline some of the sessions. As I am a backend software craftsman you may recognise easily that I skipped the marketing and lawfully oriented parts of the event and tried to focus on the more technical talks.
```

Comment:
In order to give a better impression of the actual contents of the unconference, I'd like to outline some of the sessions. As I am a backend developer, I will focus on the more technical talks and neglect the sessions about marketing or law.